app/services/rapid_ready_for_decision/views/shared/medications.erb

```diff
@@ -35,25 +40,31 @@
         <%= medication['dosageInstructions'].join('; ') %><br />
       <% end %>
 
+      <%# TODO: Add test for additional fields and make sure to include additional medications %>
+
+      Prescribed on:
```

Comment:
Should the Prescribed on, Duration and Refills be nested within the conditional statement?

Reply:
According to the ticket description, no:
> medications: include label even if there is no data for route, refills, etc.

I'll let @allthesignals confirm that interpretation.

Reply:
Yep, thanks for raising this! We tried it both ways and designers preferred label + no data in those situations.

Reply:
Very nice! I'll approve once the changes are all set

Reply:
@LindseySaari thanks, Lindsey, the changes should be good to go.